src/amr_criteria/refinement_package.hpp

```diff
@@ -53,5 +55,13 @@ void SecondDerivative(const AMRBounds &bnds, MeshData<Real> *md, const std::stri
 } // namespace Refinement
 
 } // namespace parthenon
+namespace Kokkos {
+template <>
+struct reduction_identity<parthenon::AmrTag> {
+  using AmrTag = parthenon::AmrTag;
+  KOKKOS_FORCEINLINE_FUNCTION constexpr static AmrTag max() { return AmrTag::refine; }
+  KOKKOS_FORCEINLINE_FUNCTION constexpr static AmrTag min() { return AmrTag::derefine; }
```

Comment:
I think I am be confused about how `Kokkos::reduction_identity<T>` works. I thought it was supposed to provide the identity for a given operation, so in the case of `max()` for `AmrTag` I would have thought the identity would be `AmrTag::derefine` since
```c++
max(AmrTag::derefine, AmrTag::derefine) = AmrTag::derefine;
max(AmrTag::derefine, AmrTag::same) = AmrTag::same;
max(AmrTag::derefine, AmrTag::refine) = AmrTag::refine;
```
given that `AmrTag::derefine -> -1`, etc. in the enum.

Reply:
I am probably just missing something obvious though.